internal/pkg/aws/ecs/ecs.go

```diff
@@ -100,20 +101,60 @@ func (e *ECS) TaskDefinition(taskDefName string) (*TaskDefinition, error) {
 
 // Service calls ECS API and returns the specified service running in the cluster.
 func (e *ECS) Service(clusterName, serviceName string) (*Service, error) {
-	resp, err := e.client.DescribeServices(&ecs.DescribeServicesInput{
-		Cluster:  aws.String(clusterName),
-		Services: aws.StringSlice([]string{serviceName}),
-	})
+	svcs, err := e.Services(clusterName, serviceName)
 	if err != nil {
-		return nil, fmt.Errorf("describe service %s: %w", serviceName, err)
+		return nil, err
 	}
-	for _, service := range resp.Services {
-		if aws.StringValue(service.ServiceName) == serviceName {
-			svc := Service(*service)
-			return &svc, nil
+	if aws.StringValue(svcs[0].ServiceName) != serviceName {
+		return nil, fmt.Errorf("cannot find service %s", serviceName)
+	}
+
+	return svcs[0], nil
+}
+
+// Services calls the ECS API and returns all of the services running in cluster.
```

Comment:
nit nit nit super tiny nit: I _think_ by "all of the services", you probably meant `services` the parameter name for the variadic trailing arguments. But it reads like you could call `e.Services("my-cluster")` and get all the services from the clusters, while in reality we just return a `nil` slice.

```suggestion
// Services calls the ECS API and returns all of the specified services running in cluster.
```